packages/editor/src/components/post-author/index.js

```diff
@@ -28,8 +30,14 @@ function PostAuthor() {
 			const author = __unstableGetAuthor(
 				getEditedPostAttribute( 'author' )
 			);
-			const query =
-				! fieldValue || '' === fieldValue ? {} : { search: fieldValue };
+			const isSearching =
+				isCombobox &&
+				( postAuthor
+					? fieldValue && postAuthor.name !== fieldValue
```

Comment:
Why are we comparing against the  "name" here?

Reply:
This prevent's searching when the author's name matches the input value, the main reason it is here is to prevent a search when initially focusing on the field. Looks like the field now empties when focused, so I can probably remove this.